### PR TITLE
Use latest ember LTS

### PR DESCRIPTION
**Pull request by nanderss**
- Add Circle CI 2.0 configuration
- Use latest LTS  Ember.js version

### CODE REVIEW

**Comment by perlun** [comment]
@nanderss It passes on CircleCI, but fails on Travis. Should we disable Travis for this repo?

**Comment by nanderss** [comment]
No, not yet.

**Review by olleolleolle — APPROVED** [review]
Let's use this change!

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ env:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - EMBER_TRY_SCENARIO=ember-lts-2.12
-    - EMBER_TRY_SCENARIO=ember-lts-2.16
+    - EMBER_TRY_SCENARIO=ember-lts-2.18
```
Comment (mriska) [inline .travis.yml:26]: This scenario does not exist.

**Comment by mriska** [comment]
The tests that ran on CircleCI included LTS-2.16 and not LTS-2.18 since that is the scenario that was actually defined. LTS-2.18 was only mentioned in the travis matrix, and hence it failed on Travis.

**Pull request merged**